lib/Configuration.php

```diff
@@ -120,7 +124,7 @@ class Configuration
     public function __construct()
     {
         $this->tempFolderPath = sys_get_temp_dir();
-        $this->userAgent = 'brevo_clientAPI/v' . $GLOBALS['version'] . '/php'; 
+        $this->userAgent = 'brevo_clientAPI/v' . self::VERSION . '/php';
```

Comment:
Since this is the only place where the VERSION constant is used, isn't it better to just write down the full string?
```suggestion
        $this->userAgent = 'brevo_clientAPI/v2.0.0/php';
```

Reply:
I will apply your suggestion